x/gov/codec.proto

```diff
@@ -86,10 +94,16 @@ message Proposal {
   string title = 2;
   // Description of the proposal in text form.
   string description = 3;
-  // Reference to the election rule
-  bytes election_rule_id = 4 [(gogoproto.customname) = "ElectionRuleID"];
+  // ElectionRuleRef is a reference to the election rule
+  orm.VersionedIDRef election_rule_ref = 4 [
+    (gogoproto.customname) = "ElectionRuleRef",
```

Comment:
Same comment as other PR with unnecessary customname directive.

I also want to mention that these `[(gogoprotobuf....)]` directives break compilation in javascript or other languages. I [remove them with a regexp](https://github.com/iov-one/iov-core/blob/master/packages/iov-bns/scripts/cleaned_protos.sh#L27) before compiling them in js. But splitting them on multiple lines makes this harder to do. Can you just make them on one line again? Or propose a fix to the js scripts?

```shell
    # This just removes all options after the fields
    sed -ie 's/ *\[[^]]*\];/;/g' "$outfile"
    # removes illegal ;; typos
    sed -ie 's/;;/;/' "$outfile"
```

Reply:
"Fixing" sed to handle your multiline seems like it will be a hair-pulling experience:
https://unix.stackexchange.com/a/26290

Reply:
Splitting into multiple lines is done by the formatting tool we are using. Maybe it can be configured?

https://github.com/iov-one/weave/blob/master/Makefile#L61-L62

Reply:
Maybe we need to configure prototool. It seems to have a config file format:

https://github.com/uber/prototool/blob/dev/etc/config/example/prototool.yaml#L66-L75

Happy for another PR on this. I would saw this is blocking for a 0.15 release (having protobuf files we can compile in iov-core)

Reply:
😭 they got formatted into multiple lines automatically.

Reply:
Removed the naming annotations so that they are 1 liners again